src/views/Dashboard/Customer/Home/RequestTest/Checkout/PaymentCheckout.vue

```diff
@@ -48,13 +50,16 @@ export default {
     service: {}
   }),
 
-  async mounted () {
+  async mounted() {
```

Comment:
Please Update using `created` for this one

Reply:
updated